docs/content/users/providers/platform.md

```diff
@@ -42,6 +42,20 @@ web_environment:
 4. Run `ddev pull platform`. After you agree to the prompt, the current upstream database and files will be downloaded.
 5. Optionally use `ddev push platform` to push local files and database to Platform.sh. The [`ddev push`](../basics/commands.md#push) command can potentially damage your production site, so we don’t recommend using it.
 
+If you have more than one database on your Platform.sh project, you'll need to choose which one you want to use
```

Comment:
@rfay I understand only one database will be handled by the integration, but what would you recommend if I had additional databases I also wanted to pull into DDEV?

Create my own modified version of `platform.yaml` and modify [`db_pull_command`](https://github.com/drud/ddev/blob/master/pkg/ddevapp/dotddev_assets/providers/platform.yaml#L44)?

Reply:
In this version, *all* databases will be handled by the `ddev pull platform`, so line 45 is just incorrect. Thanks for catching that.

Reply:
Am I understanding correctly that all databases will be pulled, but that I still need to designate a `PLATFORM_PRIMARY_RELATIONSHIP` so DDEV knows what to use for its (default) `'db'` database?

Reply:
Ultimately asking while I attempt to improve the wording [over here](https://github.com/mattstein/ddev/commit/317f0304cea9c1a4b93ea60e2e7a598611ee22ad).

Reply:
@mattstein this is correct.

- All databases will be downloaded
- The remote database that is flagged as primary will be imported into the `db` database of DDEV (the default one)
- All other remote databases will be imported into ddev databases with the same as in the remote

I believe I've never used the word database that much in such a short message

Reply:
@bserem Thanks for clarifying the database behavior for this database-related documentation adjustment regarding databases. 😄

Reply:
Not all projects will need for a "db" database to be populated. For example, as the ddev-platformsh add-on matures, it should be able to handle databases of any number that have the same names they do on Platform.sh, and the "db" database not be used at all.

Reply:
So this is more of an *if* you have multiple databases *and* want to designate a primary to use with `'db'`, yes? If so I’ll update https://github.com/drud/ddev/pull/4469 accordingly.

Reply:
That is correct. Most people who have ever used DDEV will want that. There may be contexts in the future where that's not needed.  The same may be true for Acquia or Pantheon or other hosts, where the settings are wired to particular database names and people are *not* using DDEV's automatically generated settings, like `settings.ddev.php` or Craft CMS's .env configuration.

Reply:
Revised [in this commit](https://github.com/drud/ddev/pull/4469/commits/6a311d52db800af254c2c9f975edf7e30e362c7f), which is hopefully more accurate! 😅